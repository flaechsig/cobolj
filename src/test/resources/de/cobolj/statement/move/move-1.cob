IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-NUMBER-1 PIC 9(3) VALUES 50.
   01 WS-NUMBER-2 PIC 9(4)V9(2).

PROCEDURE DIVISION.
   MOVE 5.1 TO WS-NUMBER-1 WS-NUMBER-2
   DISPLAY WS-NUMBER-1
   DISPLAY WS-NUMBER-2
STOP RUN.