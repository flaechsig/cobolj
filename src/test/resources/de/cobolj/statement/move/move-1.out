005
0005.10
